src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/Movement.kt

```diff
@@ -8,6 +8,8 @@ import java.time.LocalDate
 @JsonInclude(NON_NULL)
 @Schema(description = "A movement into prison")
 data class Movement(
+  @Schema(description = "Movement ID", example = "0573de83-8a29-42aa-9ede-1068bc433fc5")
+  val id: String,
```

Comment:
thought i'd do the work for adding the ID into the  response here

Reply:
Wasn't there two bits to that ticket though? Also adding gender

This was going to be optional - we wouldn't set it if it was a transfer (from the prison api)